src/index-metadbs/matterverse/v1/links.json

```diff
@@ -0,0 +1,35 @@
+{
+   "meta": {
+      "api_version": "1.0.0",
+      "query": {
+         "representation": "/links"
+      },
+      "more_data_available": false,
+      "schema": "https://schemas.optimade.org/openapi/v1.0/optimade_index.json"
+   },
+   "data": [
+      {
+         "id": "matterverse",
+         "type": "links",
+         "attributes": {
+            "name": "Matterverse",
+            "description": "Database of 31 million hypothetical materials generated through advanced machine learning algorithms",
```

Comment:
I personally would not add the number ("31 million") as it may change in the future and may become out-of-sync. Maybe change to "over 31 million"?